### PR TITLE
Update ui

### PR DESCRIPTION
**Pull request by sgn00**
1. Fix truncation of question and answer
2. Add progress bar
3. Add stats ui

### CODE REVIEW

**Review by tan-zhuoyao — COMMENTED** [review]
LGTM

File: src/main/java/seedu/flashcard/ui/FlashcardQuestionCard.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public FlashcardQuestionCard(Flashcard flashcard) {
 
         if (flashcard.getRating().toString().length() > 0) {
             rating.setText(flashcard.getRating().toString());
-            ratingIcon.setText(" \uD83D\uDFCA");
+            ratingIcon.setText(" \u2605");
```
Comment (moyj01) [inline src/main/java/seedu/flashcard/ui/FlashcardQuestionCard.java]: You may consider creating an Icon class to store the commonly used Unicode characters

File: src/main/java/seedu/flashcard/ui/FlashcardStatsCard.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+     */
+    public FlashcardStatsCard(Flashcard flashcard) {
+        super(FXML);
+        this.flashcard = flashcard;
+        question.setText(flashcard.getQuestion().toString());
+        int successFrequency = flashcard.getStatistics().getSuccessFrequency();
+        int reviewedFrequency = flashcard.getStatistics().getReviewFrequency();
+        int failureFrequency = reviewedFrequency - successFrequency;
+        double successRate = ((double) successFrequency / (double) reviewedFrequency) * 100;
+        double failureRate = 100 - successRate;
```
Comment (moyj01) [inline src/main/java/seedu/flashcard/ui/FlashcardStatsCard.java]: Perhaps, you should abstract these calculations to methods

**Review by moyj01 — APPROVED** [review]
Some comments from myself. Nothing glaring in particular. Maybe you can make some changes in future iterations.

File: src/main/java/seedu/flashcard/ui/FlashcardStatsCard.java (hunk 2 above)
Comment (sgn00) [reply src/main/java/seedu/flashcard/ui/FlashcardStatsCard.java]: i think they are only used once so there is no need for that.

File: src/main/java/seedu/flashcard/ui/FlashcardQuestionCard.java (hunk 1 above)
Comment (sgn00) [reply src/main/java/seedu/flashcard/ui/FlashcardQuestionCard.java]: done

**Pull request merged**